### PR TITLE
ReversingLabs TitaniumCloud 2.0.4

### PR DESCRIPTION
**Pull request by MislavReversingLabs**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Minor changes.
Simply implemented the existing User-Agent string into the objects of the ReversingLabsTitatinumCloudv2.py script.
The rest of the changes are just version bumps so the in-code version strings match the versions displayed in the pack_metadata.json files.
Code-wise everything stays the same.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by dorschw** [comment]
Hi @MislavReversingLabs, thank you for the contribution!

Reviewing this PR will be slightly delayed as we are in the midst of the local holiday season. We'll have a look as soon as we're back at work.

**Comment by MislavReversingLabs** [comment]
@dorschw Hi. When can we expect this PR to be reviewed? Thanks!

File: Packs/ReversingLabs_A1000/ReleaseNotes/2_0_6.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### ReversingLabs A1000 v2
+- Just updated the in-code version string to 2.0.6
```
Comment (dorschw) [inline Packs/ReversingLabs_A1000/ReleaseNotes/2_0_6.md:4]:
```suggestion
- Just updated the in-code version string to 2.0.6
- Updated Docker image to demisto/reversinglabs-sdk-py3:2.0.0.34140.
```

File: Packs/ReversingLabs_TitaniumScale/ReleaseNotes/1_0_2.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### ReversingLabs TitaniumScale
+- Just updated the in-code version string to 1.0.2
```
Comment (dorschw) [inline Packs/ReversingLabs_TitaniumScale/ReleaseNotes/1_0_2.md:4]:
```suggestion
- Just updated the in-code version string to 1.0.2 
- Updated Docker image to demisto/reversinglabs-sdk-py3:2.0.0.34140.
```

File: Packs/ReversingLabs_Titanium_Cloud/ReleaseNotes/2_0_4.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### ReversingLabs TitaniumCloud v2
+- Implemented the use of the ReversingLabs User-Agent string.
```
Comment (dorschw) [inline Packs/ReversingLabs_Titanium_Cloud/ReleaseNotes/2_0_4.md:4]:
```suggestion
- Implemented the use of the ReversingLabs User-Agent string.
- Updated Docker image to demisto/reversinglabs-sdk-py3:2.0.0.34140.
```

**Review by dorschw — CHANGES_REQUESTED** [review]
Nice job, please accept the suggested changes into the release notes, and we'll be ready to merge.

**Comment by MislavReversingLabs** [comment]
@dorschw why is the build failing now after I accepted your release notes changes?

**Comment by dorschw** [comment]
Seems like an internal CI issue, we're looking into it, sorry for the inconvenience.

**Comment by dorschw** [comment]
Please pull from `demisto/master` into this branch

**Comment by MislavReversingLabs** [comment]
@dorschw can you check the PR now and approve it?

**Review by dorschw — APPROVED** [review]

**Pull request merged**